osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/config/ScheduledTaskExecutorJobConfig.java

```diff
@@ -30,13 +30,21 @@ public class ScheduledTaskExecutorJobConfig {
   @Value("${scheduling.task.page.size}")
   private int scheduledTaskPageSize;
 
+  @Value("${scheduling.task.pending.duration.max.seconds:3600}")
```

Comment:
Isn't there another time specified for the retry timeout? Shouldn't this time be related to that time?

Reply:
There is a **MaxScheduledTime** which defines until what time the task must be retried. This is a property of the ScheduledTask. This newly introduced setting defines the max duration for a pending task to keep waiting for a response. If there is no response within that duration the task will resend a request.